src/layers/PixelatedBitmapLayer.js

```diff
@@ -1,5 +1,5 @@
-import { BitmapLayer } from '@deck.gl/layers';
-import { CompositeLayer } from '@deck.gl/core';
+import { BitmapLayer } from '@deck.gl/layers'; // eslint-disable-line import/no-extraneous-dependencies
+import { CompositeLayer } from '@deck.gl/core'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
are these errors because `@deck.gl/*` isn't explicitly listed in deps, just `deck.gl`? Might be fair to ignore all `@deck.gl/*` imports

Reply:
not sure if this is something we could add to the eslint config.